apps/metadata-editor/src/app/edit/components/publish-button/publish-button.component.ts

```diff
@@ -42,9 +48,48 @@ export class PublishButtonComponent {
     })
   )
 
-  constructor(private facade: EditorFacade) {}
+  record$ = this.facade.record$
+  activeUser$: Observable<UserModel>
+
+  constructor(
+    private facade: EditorFacade,
+    private recordsApiService: RecordsApiService,
+    private platformService: PlatformServiceInterface,
+    private groupApiService: GroupsApiService
+  ) {
+    this.activeUser$ = this.platformService.getMe()
```

Comment:
Can't we call directly `this.platformService.getMe()` where it's neded. the `activeUser$` variable doesn't bring any added value, does it?